src/commands/deploy/deploy.ts

```diff
@@ -285,7 +285,7 @@ async function deployToSas9(
   }
 }
 
-async function deployToSasjs(target: Target, sasjs?: SASjs) {
+async function deployToSasjs(target: Target, isLocal: boolean, sasjs?: SASjs) {
```

Comment:
can we get a JSDoc explanation here to explain what `isLocal` means / does ?

Reply:
provided explanation through JSDoc comments.